website/docs/r/app_service_plan.html.markdown

```diff
@@ -0,0 +1,65 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_app_service_plan"
+sidebar_current: "docs-azurerm-resource-app_service_plan"
+description: |-
+  Create an App Service Plan component.
+---
+
+# azurerm\_app\_service\_plan
+
+Create an App Service Plan component.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "test" {
+  name     = "api-rg-pro"
+  location = "West Europe"
+}
+
+resource "azurerm_app_service_plan" "test" {
+  name                = "api-appserviceplan-pro"
+  location            = "West Europe"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  sku {
+    tier = "Standard"
+    size = "S1"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of the App Service Plan component. Changing this forces a
+    new resource to be created.
+
+* `resource_group_name` - (Required) The name of the resource group in which to
+    create the App Service Plan component.
+
+* `location` - (Required) Specifies the supported Azure location where the resource exists. Changing this forces a new resource to be created.
+
+`sku` supports the following:
+
+* `tier` - (Required) Specified the plan's pricing tier.
+* `size` - (Required) Specified the plan's instance size. 
+
+* `maximum_number_of_workers` - (Optional) .
```

Comment:
could we add a description for this field, here's the description from the Azure docs: "Maximum number of instances that can be assigned to this App Service plan."